packages/rancher-istio-1.10/charts/Chart.yaml

```diff
@@ -13,7 +13,7 @@ annotations:
   catalog.cattle.io/release-name: rancher-istio
   catalog.cattle.io/ui-component: istio
   catalog.cattle.io/provides-gvr: networking.istio.io.virtualservice/v1beta1
-  catalog.cattle.io/auto-install: rancher-kiali-server-crd=1.32.100-rc01
+  catalog.cattle.io/auto-install: rancher-kiali-server-crd=1.35.1
```

Comment:
@brendarearden can you double check that, when adding your fork as a Chart Repository in Rancher's UI, the Istio chart triggers the CRD install as expected?

It should be the case that this works, but I'm wondering whether our auto-install logic handles the fact that it should ignore the build metadata (e.g. +up) when trying to find a chart with this version string.

Reply:
`This chart requires rancher-kiali-server-crd=1.35.1 but no matching chart was found` When I attempted to install it in the UI

Reply:
@aiyengar2 I went ahead and updated the auto-install to include the metadata at this time, https://github.com/rancher/dashboard/issues/3206 << issue opened here to correct it in the ui so that future charts do not need to include the metadata